oak_functions/examples/hello_world/module/src/lib.rs

```diff
@@ -16,31 +16,42 @@
 
 //! Oak functions hello world example. Responds with `Hello $request_body!` to every request.
 
+#![feature(try_blocks)]
+
+use serde::Deserialize;
+
 #[cfg(test)]
 mod tests;
 
+#[derive(Deserialize)]
+struct Location {
```

Comment:
You can do this in order to make custom field names:
```rust
    #[serde(rename = "lon")]
    longitude: f32,
    #[serde(rename = "lat")]
    latitude: f32,
```

Reply:
Done.